src/styles/testribbon.scss

```diff
@@ -0,0 +1,76 @@
+.corner-ribbon .wrapper {
+  background: transparent;
+  height: 140px;
+  width: 35px;
+  position: absolute;
+  bottom: -34px;
+  right: 19px;
+  transform: rotate(45deg);
+  overflow: hidden;
+  z-index: 32;
+  -webkit-backface-visibility: hidden;
+
+  .container {
+    background : transparent;
+    height: 110px;
+    width: 110px;
+    position: absolute;
+    left: -54px;
+    top: 15px;
+    overflow: hidden;
+    transform: rotate(45deg);
+    
+    &:before { /* Top Curl */ 
+      content: "";
+      display: block;
+      position: absolute;
+      right: 94px;
+      top: 0px;
+      width: 0;
+      height: 0;
+      border-left: 6px solid transparent;
+      border-right: 6px solid transparent;
+      border-bottom: 6px solid #b37943;
+    }
+
+    &:after { /*  Bottom Curl */ 
+      content: "";
+      display: block;
+      position: absolute;
+      right: 0;
+      top: 92px;
+      width: 0;
+      height: 0;
+      border-top: 6px solid transparent;
+      border-bottom: 6px solid transparent;
+      border-left: 6px solid #b37943;
+    }
+
+    .ribbon {
+      width: 140px;
+      height: 21px;
+      position: relative;
+      top: 32px;
+      right: 3px;
+      z-index: 1;
+      overflow: hidden;
+      transform:rotate(45deg);
+      background: #ffb46e;
```

Comment:
Should use a variable with the color. Perhaps refactor the scss a bit to make a `variables.scss` that `main.scss` and `fullcalendar.scss` also imports?

Reply:
fantastic idea